### ISSUE TITLE
CKAN accessibility

### ISSUE THREAD

**Issue by maryLoi** [hasadna/datacity-k8s#64]
reference from Zomer
https://github.com/OriHoch/data4dappl/issues?q=is%3Aissue+label%3AAccessibility+is%3Aclosed

- [ ] image overlay text
לכל תמונה או לוגו יהיה טקסט מרחף עם תיאור

- [ ] zoom buttons
פתרון אפשרי הוא כפתורים של פלוס ומינוס לשינוי גודל הפונט, כמו במידע לעם
https://github.com/OriHoch/data4dappl/issues/101

- [ ] color contrast
can use a plugin to check colors
https://webaim.org/resources/contrastchecker/

**Comment by niryariv** [comment]
@OriHoch is there anything we can do now for above?

**Comment by OriHoch** [comment]
need to define this in more details - which texts should appear, where we get them from, how zoom buttons should appear, where, which colors to use etc..

**Comment by niryariv** [comment]
Agreed. Assigning self as design tasks. 

Is there any CKAN extension etc to detect images missing alt text in the templates? (outside of external tools)

**Comment by OriHoch** [comment]
no, there isn't such an extension

**Comment by niryariv** [comment]
(note to self: https://wave.webaim.org/report#/https://maale-adummim.datacity.org.il/ )

**Comment by niryariv** [comment]
closing in favor of #79
